adserver/models.py

```diff
@@ -365,6 +365,18 @@ def __str__(self):
     def ad_count(self):
         return Advertisement.objects.filter(flight__campaign=self).count()
 
+    def allowed_ad_types(self, exclude_deprecated=False):
+        """Get the valid ad types for this campaign."""
+        queryset = AdType.objects.filter(
+            models.Q(publisher_groups=None)
+            | models.Q(  # Global ad types across all publishers
```

Comment:
Isn't this comment talking about `publisher_groups=None`?

Reply:
This is awkward. This actually does apply above. I think it got formatted like this. I'll see if I can re-format it.